src/main/java/com/kodilla/ecommercee/ProductController.java

```diff
@@ -1,4 +1,24 @@
 package com.kodilla.ecommercee;
 
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/v1/product")
 public class ProductController {
+
+    @GetMapping
+    public void get(){}
+
+    @GetMapping({"/{id}"})
+    public void get(@PathVariable Long id){}
+
+    @PostMapping
+    public void create(){}
+
+    @PutMapping("/{product}")
+    public void update(@PathVariable String product){}
```

Comment:
raczej update polega na tym, ze podajemy obiekt dto do zupdatowania

Reply:
Tak samo jak wyżej napisałem, nie mamy w projekcie jeszcze takich encji, więc nie mogę ich dać jako parametr.

Reply:
Chyba, że mogę je po prostu stworzyć, to wtedy zmienia postać rzeczy ;)

Reply:
j.w.